### DISCUSSION TITLE
PRC 163 dual channel radio

### DISCUSSION THREAD

**Discussion by Obscur4ns** [IDI-Systems/acre2#1314] [Ideas]
Is it possible? And if so, any plans to add any dual channel radios? Obviously TFAR can do it but it's inferior

**Comment by jonpas** [comment]
It is possible, although we don't have any yet so it might need some additional work.

We would be happy to see contributions of dual-channel radios for sure!

**Comment by Obscur4ns** [comment]
Amazing, any insight on what work and where I can look for making an attempt at it?

**Reply by jonpas** [reply]
It is best to look at one of the pull requests that added a radio. Dual-channel setup should mostly be changes to the radio code itself and then ducking for dual reception.
